### PR TITLE
completed css-fonts exercise, avallon smith

### PR DESCRIPTION
**Pull request by avallonsmith**
image

Fonts: 
-Romanesco
-Oswald

### CODE REVIEW

File: css-fonts/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+  href="https://fonts.googleapis.com/css2?family=Cinzel+Decorative&family=Press+Start+2P&family=Satisfy&display=swap"
+  rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Cinzel+Decorative&family=Press+Start+2P&family=Satisfy&display=swap"
+    rel="stylesheet">
```
Comment (TimelikeClosure) [inline css-fonts/index.html:17]: You should only need to include one copy of the Google Font stylesheet, not two.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: css-fonts/index.html (hunk 1 above)
Comment (avallonsmith) [reply css-fonts/index.html]: image

File: css-fonts/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Cinzel+Decorative&family=Press+Start+2P&family=Romanesco&family=Satisfy&display=swap"
+    rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Cinzel+Decorative&family=Press+Start+2P&family=Satisfy&display=swap"
+    rel="stylesheet">
```
Comment (TimelikeClosure) [inline css-fonts/index.html:17]: You still have duplicate `<link>`s here. There only needs to be one copy of each `rel="preconnect"` one. The first `rel="stylesheet"` contains the `Romenesco` font-family (along with a few others), but neither one contains the `Oswald` font-family.

---

image

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: css-fonts/index.html (hunk 2 above)
Comment (avallonsmith) [reply css-fonts/index.html]: image

Duplicates have been removed, links have been properly updated

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**